crates/router/src/core/payments/routing.rs

```diff
@@ -170,7 +170,111 @@ pub fn make_dsl_input_for_payouts(
 pub fn make_dsl_input(
     payments_dsl_input: &routing::PaymentsDslInput<'_>,
 ) -> RoutingResult<dsl_inputs::BackendInput> {
-    todo!()
+    let mandate_data = dsl_inputs::MandateData {
+        mandate_acceptance_type: payments_dsl_input.setup_mandate.as_ref().and_then(
+            |mandate_data| {
+                mandate_data.customer_acceptance.clone().map(
```

Comment:
Do we need the `clone()` here? (Can change in v1 code as well, if using `as_ref()` works.)

```suggestion
                mandate_data.customer_acceptance.as_ref().map(
```